src/material/datepicker/datepicker-content.html

```diff
@@ -1,4 +1,7 @@
-<div cdkTrapFocus>
+<div
+  cdkTrapFocus
+  class="mat-datepicker-content-wrapper"
```

Comment:
FWIW I tend to avoid using "wrapper" for CSS classes since it's more "implementation detail" rather than anything semantically meaningful, but I don't have a better suggestion in this instance.

Reply:
Changed it to "container", but it isn't much better.